crates/biome_js_formatter/src/ts/bindings/type_parameters.rs

```diff
@@ -25,18 +30,26 @@ impl FormatNodeRule<TsTypeParameters> for FormatTsTypeParameters {
             l_angle_token,
         } = node.as_fields();
 
-        if items.is_empty() {
+        if items.is_empty() && self.options.is_type_or_interface_decl {
+            write!(
+                f,
+                [&format_args![
+                    l_angle_token.format(),
+                    r_angle_token.format()
+                ]]
```

Comment:
this `&format_args![]` can be removed, since it's just being passed directly to `write!`, which takes an array of arguments already. `format_args!` is only needed to bundle up multiple elements as a single argument for a function, like `group` or `indent`.
```suggestion
                [
                    l_angle_token.format(),
                    r_angle_token.format()
                ]
```